PURL-TYPES.rst

```diff
@@ -582,6 +582,50 @@ swift
       pkg:swift/github.com/Alamofire/Alamofire@5.4.3
       pkg:swift/github.com/RxSwiftCommunity/RxFlow@2.12.4
 
+
+YOCTO Linux
+-----------
+``yocto`` for Yocto Linux recipes:
+
+There is no default package repository. The classifier `repository_url` is optional.
+
+:namespace: The namespace is the name of the layer which provides the recipe. The layer name needs
+to match with the name used in the `meta-<namespace>/conf/layer.conf`.
```

Comment:
`meta-<namespace>` is a common convention to name layers when users download them, but is not mandatory. The better way to describe this would be:

```
The layer name as specified in the `BBFILE_COLLECTIONS` variable in `conf/layer.conf` of the layer.
```

Reply:
Done!